src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/IImmutableStack.cs

```diff
@@ -2,14 +2,15 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections.Generic;
-using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
 
 namespace System.Collections.Immutable
 {
     /// <summary>
     /// An immutable stack.
     /// </summary>
     /// <typeparam name="T">The type of elements stored in the stack.</typeparam>
+    [CollectionBuilder(typeof(ImmutableStack), nameof(ImmutableStack.Create))]
```

Comment:
Guessing test coverage is something we'll be adding once we get collection literals support from the compiler?

Reply:
Yeah. There's not really anything to test right now.